GreenhouseNode/GreenhouseNode/controllers/WServo.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * WServo.c
+ *
+ * Created: 12/14/2021 11:39:18 AM
+ *  Author: tolde
+ */ 
+
+#include <ATMEGA_FreeRTOS.h>
+#include <rc_servo.h>
+#include "WServo.h"
+
+typedef struct direction{
+	int8_t direction_servo;
+	int8_t directionToDestroy;
+}direction;
+
+void initialise_servo()
+{
+	rc_servo_initialise();
+}
+
+void open_servo()
+{
+	rc_servo_setPosition(0, 100);
+}
+
+void close_servo()
+{
+	rc_servo_setPosition(0, -100);
+}
+
+void set_servo(uint8_t portNo, direction_t position)
+{
+	portNo = 0;
+	position->direction_servo;
```

Comment:
What is this code supposed to do? You have no way of creating a `direction_t` object and no way of setting the servo position directly.

Why don't you just `rc_servo_setPosition(portNo, position);` do we care about saving the position here since I assume we already have it in the configuration class?